Source/WebCore/Modules/indexeddb/server/UniqueIDBDatabaseTransaction.cpp

```diff
@@ -117,34 +117,28 @@ bool UniqueIDBDatabaseTransaction::isReadOnly() const
     return m_transactionInfo.mode() == IDBTransactionMode::Readonly;
 }   
 
-void UniqueIDBDatabaseTransaction::commit(uint64_t pendingRequestCount)
+bool UniqueIDBDatabaseTransaction::shouldAbortDueToUnhandledRequestError(uint64_t handledRequestResultsCount)
+{
+    if (handledRequestResultsCount > m_requestResults.size()) {
+        RELEASE_LOG_ERROR(IndexedDB, "%p - UniqueIDBDatabaseTransaction::shouldAbortDueToUnhandledRequestError: finished request count (%" PRIu64 ") is bigger than total request count %zu", this, handledRequestResultsCount, m_requestResults.size());
+        return false;
+    }
+
+    auto pendingRequestResults = m_requestResults.subvector(handledRequestResultsCount);
```

Comment:
We probably want to use `subspan()` instead of `subvector()`? Constructing a new vector seems unnecessarily expensive here.